apps/src/p5lab/AnimationTab/AnimationList.jsx

```diff
@@ -56,6 +72,12 @@ class AnimationList extends React.Component {
       </ScrollableList>
     );
   }
+
+  backgroundCategoryAnimations(key) {
+    return (this.props.animationList.propsByKey[key].categories || []).includes(
```

Comment:
`this.props.animationList.propsByKey[key].categories` will throw an error at various points if an object we expect to be available is undefined. we can use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) to protect against that

Reply:
Thanks! Going to change to `this.props.animationList.propsByKey?.[key]?.categories`
We should be able to assume `this.props.animationList` is defined because it's required by the component, right?